go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/go-mysql-org/go-mysql v1.5.0 // replaced
 	github.com/go-piv/piv-go v1.10.0
 	github.com/go-redis/redis/v9 v9.0.0-rc.1 // replaced
-	github.com/go-webauthn/webauthn v0.5.0
```

Comment:
revert

Reply:
i have also added it to exclusions

Reply:
I am not sure, they just did another breaking change in last week or two: https://github.com/go-webauthn/webauthn/releases. Maybe we can wait week or two and check if this is more stable in order not to apply multiple fixes.